src/pint/models/model_builder.py

```diff
@@ -64,6 +65,20 @@ def parse_parfile(parfile):
     return parfile_dict
 
 
+def _replace_fdjump_in_parfile_dict(pardict):
```

Comment:
Can you add a docstring to explain what this function does (and why it's needed)?

Reply:
added.